screens/map/ProductsScreen.js

```diff
@@ -1,40 +1,52 @@
 import React from 'react';
-import { FlatList } from 'react-native';
+import { FlatList, View } from 'react-native';
 import { ScrollView } from 'react-native-gesture-handler';
 import ProductCard from '../../components/product/ProductCard';
-import { styles } from '../../styled/product';
+import {
+  ProductListHeaderContainer,
+  ProductListContainer,
+  ProductListTitle,
+  styles
```

Comment:
Now that `FlatList` is removed, we don't need to import `styles` from `/styled/product`